### ISSUE TITLE
Old files are not showing

### ISSUE THREAD

**Issue by skullmax** [jpillora/cloud-torrent#231]
After adding a torrent with lots of files and folders, older files/folders are not showing on Desktop browsers. It crashes on smartphone browsers.

**Comment by jpillora** [comment]
Yep, doesn't handle many files very well. Lazy loading the file list would
be quite a large change, requiring directory pub/sub, and more. A lot of
effort involved. Time I don't have unfortunately.
